src/components/customplaybackrate/customplaybackrate.js

```diff
@@ -0,0 +1,91 @@
+import { playbackManager } from '../playback/playbackmanager';
```

Comment:
Not sure, maybe we should use TypeScript for new components?
Ideally, add React components, but this can be problematic for use in non-React parts of the UI.